tools/playground/src/main.rs

```diff
@@ -24,9 +27,13 @@ mod extras;
 mod playground;
 
 fn main() {
-    tracing_subscriber::fmt()
-        .with_max_level(Level::DEBUG)
-        .init();
+    let mut args = env::args().skip(1);
+    let log_level = match (args.next().as_deref(), args.next()) {
+        (Some("-l" | "--log"), Some(level)) => Level::from_str(&level).unwrap_or(Level::DEBUG),
+        _ => Level::DEBUG,
+    };
```

Comment:
I'd prefer if we went ahead and pulled in clap. We'll probably want to add some additional args for things like connection mode in the future.

Go ahead and create a `struct Args` and use the clap derive api for this.